### ISSUE TITLE
Yarn package support

### ISSUE THREAD

**Issue by bender404** [FountainJS/generator-fountain-webapp#174]
Could you add support for Yarn package tools? It's much faster then NPM. I thing Yarn option in fountain js, can be in future. Thank you.

**Comment by ghost** [comment]
Agreed, to the extent that people use node_modules in their workflow (whatever they're using to install packages), it would be easier to get signed package information if the tool already did that.

**Comment by zckrs** [comment]
I like idea but this will double the number of combinaison for our tests suites.

But really I like and this is not difficult. Contributions are very appreciate

**Comment by aryehbeitz** [comment]
why not just run `yarn` instead of `npm install`?
